README.md

```diff
@@ -1,8 +1,73 @@
 # Clayground
-Sandbox environment for starting Ceramic ecosystem locally.
+
+Sandbox environment for starting Ceramic ecosystem locally. Components included:
+
+* [ipfs](https://github.com/ceramicnetwork/js-ipfs-daemon) - IPFS instance with [dag-jose](https://github.com/ceramicnetwork/js-dag-jose) codec enabled
+* [ganache](https://www.trufflesuite.com/ganache) - local Ethereum blockchain
+* [cas](https://github.com/ceramicnetwork/ceramic-anchor-service) - Typescript Ceramic Anchor Service implementation
+* [ceramic](https://github.com/ceramicnetwork/js-ceramic). Typescript Ceramic implementation.
+
+### Prerequisites
+
+In order to use the Clayground you need to install:
+
+* [Docker](https://docs.docker.com/get-docker/)
+* [docker-compose](https://docs.docker.com/compose/install/).
+
+### Running
+
+The default Clayground configuration is stored in `.env` file.
+
+Starting all the components locally in a synchronous manner:
+
+```shell script
+docker-compose up
+```
+
+For more `docker-compose` options and start configuration visit the [official documentation](https://docs.docker.com/compose/reference/overview/).
+
+Clayground has local CLI script which connects to deployed Ceramic instance.
+
+For example, creating a tile:
+```shell script
+./bin/ceramic.sh create tile '{"hello": "ceramic"}'
+```
+
+The script will launch a container per request. This can be omitted by installing the compatible CLI from [npm](https://www.npmjs.com/package/@ceramicnetwork/ceramic-cli).
```

Comment:
I know very little about using containers, but my impression was that they're usually not for such short-lived things as individual requests?  Is there a reason we wouldn't install the ceramic-cli by default, put it in the bin directory, and encourage that as the primary way to interact with the clayground?

Reply:
I wrapped this one like this because of the `sandbox` experience - the only thing you need to install is `Docker` and `docker-compose`.

We can add more information of how to build the `js-ceramic` repository and invoke it directly.

Reply:
Huh, I guess I had assumed docker could handle doing all that for you.  But maybe I just don't know enough about containers.  Anyway, if you think this way makes the most sense then that's enough for me!

Reply:
I assume that everybody's going to install the CLI or build it from the code. Hm, we can always start the container for the CLI and hop into it but I think this is better for the first try.

Reply:
I changed scripts to build `js-ceramic` and use that build one for running.